### ISSUE TITLE
Use Start/End instead of Left/Right in predictive back SwipeEdge

### ISSUE THREAD

**Issue by alexzhirkevich** [arkivanov/Decompose#534]
It will make RTL support much easier

**Comment by arkivanov** [comment]
Thanks for raising! Could you point out actual places in the code?

**Comment by alexzhirkevich** [comment]
I mean rename enum constants and support rtl. Now if i want to use only start swipe edge, I need to enable both edges and manage them in modifiers instead of just allow start in overlay. 

Or with less affort do that:
```kotlin
val isRtl = LocalLayoutDirection.current == LayoutDirection.Rtl

PredictiveBackGestureOverlay(
    backDispatcher = backDispatcher,
    backIcon = null,
    modifier = modifier,
    swipeEdges = setOf(
          if (isRtl)
              BackEvent.SwipeEdge.RIGHT
          else BackEvent.SwipeEdge.LEFT
      ),
    ),content = content
)
```

**Comment by arkivanov** [comment]
Thanks. I doubt it is possible and reasonable. The enum namings are coming from AndroidX [BackEventCompat](https://developer.android.com/reference/androidx/activity/BackEventCompat). But I will think about it.

**Comment by alexzhirkevich** [comment]
> The enum namings are coming from AndroidX [BackEventCompat](https://developer.android.com/reference/androidx/activity/BackEventCompat).

Didn't know that. Probably it really isn't reasonable. Found a small issue with code above, fixed in #535
